code/game/turfs/simulated/minerals.dm

```diff
@@ -59,11 +59,20 @@
 		to_chat(user, "<span class='notice'>You start picking...</span>")
 		P.playDigSound()
 
-		if(do_after(user,P.digspeed, target = src))
-			if(ismineralturf(src))
-				to_chat(user, "<span class='notice'>You finish cutting into the rock.</span>")
-				gets_drilled(user)
-				SSblackbox.add_details("pick_used_mining","[P.type]")
+		var/obj/item/organ/alcoholvessel/dwarf //check to see if the digger has the dwarf organ for increased mining speed
```

Comment:
@optimumtact so you'd rather access a dwarf only organ here? as opposed to a generic variable?